src/MICore/LinuxNativeMethods.cs

```diff
@@ -13,8 +13,8 @@ internal class LinuxNativeMethods
         [DllImport(Libc, EntryPoint = "kill", SetLastError = true)]
         internal static extern int Kill(int pid, int mode);
 
-        [DllImport(Libc, EntryPoint = "mkfifo", SetLastError = true)]
-        internal static extern int MkFifo(string name, int mode);
+        [DllImport(Libc, EntryPoint = "mkfifo", SetLastError = true, ThrowOnUnmappableChar = true)]
+        internal static extern int MkFifo([MarshalAs(UnmanagedType.LPWStr)]string name, int mode);
```

Comment:
I can suppress it instead of changing what the code actually does (which might be better since this is non-windows)